openapi/integrator/merchant/partner.yaml

```diff
@@ -134,6 +134,11 @@ components:
           $ref: '../components.yaml#/components/schemas/merchantReference'
         status:
           $ref: '#/components/schemas/PaymentStatus'
+        responseCode:
+          type: string
+          maxLength: 2
+          description: >-
+            Direct response code as it was gotten from Baltus
```

Comment:
How about "Response code from issuer or processor when payment was declined"?

Reply:
added "or processed" phrase as code will be added later also for approved authorisation

Reply:
"processed" is ambigous. "Response code from issuer or processor when payment was declined or approved" seems better to me. This phrasing corresponds perfectly with the statuses `AuthorisationDeclined` and `AuthorisationApproved`, then.

Reply:
Perhaps better to say "Response code from issuer or processor when payment status is AuthorisationApproved or AuthorisationDeclined".